### ISSUE TITLE
Query "buckets()" returns empty list when debug is enabled

### ISSUE THREAD

**Issue by meeusen** [influxdata/influxdb-client-python#480]
__Steps to reproduce:__
```
with InfluxDBClient(url=DB_URL, token=DB_TOKEN, org=DB_ORG, timeout=60000, debug=True) as client:
    query_api = client.query_api()
    tables = query_api.query("buckets()")
    assert len(tables) == 1, "This should return 1 table."
```

__Expected behavior:__
With v1.30.0, I get a list with 1 FluxTable: 
`[<FluxTable: 7 columns, 8 records>]`

__Actual behavior:__
With v1.31.0, I always get an empty list. When using debug=False for InfluxDBClient, then I do get the list with 1 FluxTable.

__Specifications:__
 - Client Version: 1.31.0
 - InfluxDB Version: 2.1.1
 - Platform: Linux (Fedora 36)

**Comment by bednar** [comment]
Hi @meeusen,

thanks for using our client. 

The issue is caused by https://github.com/influxdata/influxdb-client-python/pull/462: 

- https://github.com/influxdata/influxdb-client-python/blob/161e202d8b9784ed05fcf8ba49da2060910c9265/influxdb_client/_sync/rest.py#L254

I will take a look and prepare fixed version ASAP.

Regards

**Comment by bednar** [comment]
Hi @meeusen,

I have prepared a fixed version within #483. If you would like to use this fixed version before regular release, please install client via:

```
pip install git+https://github.com/influxdata/influxdb-client-python.git@exhausted_response
```

Regards
